litestar/types/serialization.py

```diff
@@ -25,17 +25,26 @@
     from pydantic import (
         BaseModel,
         ByteSize,
-        ConstrainedBytes,
-        ConstrainedDate,
         NameEmail,
-        SecretField,
         StrictBool,
     )
     from pydantic.color import Color
```

Comment:
Shouldnt this be the extra_types package now?
```suggestion
    from pydantic_extra_types.color import Color
```

Reply:
Not for v1. This package is V2 only.